app/invocation/invocation_model.tsx

```diff
@@ -22,6 +25,8 @@ export default class InvocationModel {
   invocations: invocation.Invocation[] = [];
   cacheStats: cache.CacheStats[] = [];
   scoreCard: cache.IScoreCard;
+  botSuggestions: string[] = [];
+  onChange: Subject<string> = new Subject();
```

Comment:
nit: if you omit `<string>` would that let you do `onChange.next()` instead of passing an empty string?

Reply:
Had to do <void> but that works!